docs/install/installing-istio.md

```diff
@@ -244,6 +244,22 @@ helm template --namespace=istio-system \
 kubectl apply -f istio-local-gateway.yaml
 ```
 
+Alternatively, if you want to install cluster local gateway for **development purpose**, run below command 
```

Comment:
@samodell I accidentally lost Richie's  suggestion. Could you let me know how the change in this file looks?